onnxruntime/test/providers/cpu/model_tests.cc

```diff
@@ -989,8 +993,7 @@ ::std::vector<::std::basic_string<ORTCHAR_T>> GetParameterStrings() {
       ORT_TSTR("Candy"),
       ORT_TSTR("SSD"),
       ORT_TSTR("ResNet101_DUC_HDC-12"),
-      ORT_TSTR("YOLOv3-12")
-      };
+      ORT_TSTR("YOLOv3-12")};
```

Comment:
the } means the end of broken_tests. should it be put in a new line?

Reply:
clang-format will move the '}' up unless there's a trailing ',' on the last line. 

can add a trailing ',' but why does it matter which line the '}' is on?

Reply:
I see. thank you.